app/controllers/devise_token_auth/sessions_controller.rb

```diff
@@ -30,17 +30,9 @@ def create
       end
 
       if @resource and valid_params?(field, q_value) and @resource.valid_password?(resource_params[:password]) and (!@resource.respond_to?(:active_for_authentication?) or @resource.active_for_authentication?)
-        # create client id
-        @client_id = SecureRandom.urlsafe_base64(nil, false)
-        @token     = SecureRandom.urlsafe_base64(nil, false)
-
-        @resource.tokens[@client_id] = {
-          token: BCrypt::Password.create(@token),
-          expiry: (Time.now + DeviseTokenAuth.token_lifespan).to_i
-        }
-        @resource.save
-
-        sign_in(:user, @resource, store: false, bypass: false)
+        auth = @resource.create_new_auth_token
+        @client_id = auth['client']
+        @token = auth['access-token']
```

Comment:
It's not exactly clear but they are used here.  https://github.com/weddingwire/devise_token_auth/blob/master/app/controllers/devise_token_auth/concerns/set_user_by_token.rb#L79

I noticed when I removed them the access-token was no longer being placed in the header.